libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts

```diff
@@ -156,11 +167,101 @@ export const DocumentProviderOnboarding: Form = buildForm({
           name: m.confirmationTitle,
           description: m.confirmationSubTitle,
           children: [
+            buildDividerField({
+              name: m.applicantSection.defaultMessage,
+              color: 'currentColor',
+            }),
+            buildTextField({
+              id: 'applicant.nationalId',
+              name: m.applicantNationalId,
+              width: 'half',
+              format: '######-####',
+              placeholder: '000000-0000',
+            }),
+            buildTextField({
+              id: 'applicant.name',
+              name: m.applicantName,
+              width: 'half',
+            }),
+            buildTextField({
+              id: 'applicant.address',
+              name: m.applicantAddress,
+              width: 'half',
+            }),
+            buildTextField({
+              id: 'applicant.zipCode',
+              name: m.applicantZipCode,
+              width: 'half',
+            }),
+            buildTextField({
```

Comment:
Was there a decision regarding which inputs should be half width ? I could imagine the email input can be to long for half width.

Reply:
We decided to make all the inputs full width :)